Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/AnimationPropertyInfo.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    /// <summary>
+    /// <see cref="Windows.UI.Composition.AnimationPropertyInfo"/>
+    /// </summary>
+    public class AnimationPropertyInfo
+    {
+        private Windows.UI.Composition.AnimationPropertyInfo UwpInstance { get; }
```

Comment:
If this is a private get-only property, shouldn't it be a field?

Reply:
done